Porting/updateAUTHORS.pl

```diff
@@ -0,0 +1,639 @@
+#!/usr/bin/env perl
+package Porting::updateAUTHORS;
+use strict;
+use warnings;
```

Comment:
let's be crazy and consider using function signature too :-)

Reply:
I don't really like function signatures and I don't use them in code I write myself. I don't think we need to use every feature in perl just because they are there. Also I would like this script to "just work" with whatever perl happens to be installed. There is a chicken and egg problem with some of our Perl based build tooling, sometimes people want to run it from older perls than the perl they are building, and I think those tools should just work if possible.

Reply:
interesting feedback, I was mainly teasing you 
personally I love using function signature: code shorter, and helps catching some incorrect calls